sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -339,117 +238,71 @@ public void ExplodedGraph_BothBranchesVisited()
                     }
                 };
 
-            explodedGraph.Walk();
+            // Number of Exit blocks:
```

Comment:
Minor clarification:
```suggestion
            // Number of exit block is still 1 in this case:
```